ifrs17/Import/Importers.ipynb

```diff
@@ -1059,7 +1151,14 @@
                 "\n    var mainArgs = (await GetArgsFromMainAsync<PartitionByReportingNodeAndPeriod>(dataSet)) with {ImportFormat = ImportFormats.Cashflow};",
                 "\n    if(Activity.HasErrors()) return Activity.Finish();",
                 "\n",
-                "\n    var targetArgs = await GetTargetArgsAsync(mainArgs);",
```

Comment:
I would maintain the GetTargetArgsAsync and have inside a switch case statement (new syntax) depending of the ImportFormat to deal with the different implementations.